### DISCUSSION TITLE
Need Help to compute the kp and kv value theoretically for the Position and velocity servos ?

### DISCUSSION THREAD

**Discussion by rishabkr29** [google-deepmind/mujoco#2009] [Asking for Help]
Hi,

I'm a student and I'm trying to use MuJoCo for my thesis. 

I'm looking for some help with the computation of kp and kv values theoretically for the position and velocity servo.

**Comment by yuvaltassa** [comment]
What do you mean "theoretically"?

Do you have a real servo that you want to identify?

**Comment by rishabkr29** [comment]
By saying theoretically
I mean that, the Position Servo at a slider joint which I am using as an actuator and giving my control input as position to it. so the system is failed to achieve to the desired position and we need to tuned the values of kp and kv, I wanted to understand the mathematical modelling behind the actuator section so that I can mathematically estimate the right values of kp and kv ?

I am using brushless DC motor for my hardware, and when we are using a Position servo what does it means ? What I understood is a Motor/actuator which is taking input as position and computing force applied for forward dynamics.

Here is the  response which I got last time #1304. I read the documentation it is no where mention how the actuator is being using the control input.
